_playwright-tests/auth.setup.ts

```diff
@@ -23,7 +24,27 @@ setup.describe('Setup Authentication States', async () => {
     expect(() => throwIfMissingEnvVariables()).not.toThrow();
   });
 
-  setup('Authenticate Read-Only User and Save State', async ({ page }) => {
+  setup('Authenticate rhel-operator user and save state', async ({ page }) => {
+    setup.setTimeout(60_000);
+
+    await closePopupsIfExist(page);
```

Comment:
this should be removed, the close popups helper isn't needed during auth and it can break the logout step and cause flakiness 🗑️